frontend/src/components/eMIB/Email.jsx

```diff
@@ -40,6 +40,9 @@ const styles = {
   }
 };
 
+// used to allow or prevent focusing on action-views div (see componentWillUpdate() function)
+let TABINDEX = "-1";
```

Comment:
This seems like a variable that should be attached to each _instance_ of a component, so it should probably be in state instead of global. 

Globals are great for constants, but using `let` generally indicates it should be within the class.

Reply:
I tried to use state instead of a global variable, but I was getting the **maximum update depth exceeded**. If there is a way to make it work, I'm willing to do the needed changes.